ui/components/app/simulation-preview/fiat-display.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { EtherDenomination } from '../../../../shared/constants/common';
+import {
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import CurrencyDisplay from '../../ui/currency-display/currency-display.component';
```

Comment:
Very minor, but can we import just the directory as we provide the `index.js`?

Reply:
Done... and removed this dependency.